app/models/smart_proxy.rb

```diff
@@ -88,12 +86,20 @@ def has_feature?(feature)
     self.features.any? { |proxy_feature| proxy_feature.name == feature }
   end
 
-  private
+  def status
```

Comment:
since github ate my comment again, I'll paste it here again:
```ruby
def statuses
  return @statuses if @statuses

  @status = {}
  features.each do |feature|
    name = feature.name.delete(' ')
    if (status = ProxyStatus.find_status_by_humanized_name(name))
      @statuses[name.to_sym] = status.new(self)
    end
  end
  @statuses[:version] = ProxyStatus::Version.new(self) }

  @statuses
end
```

not the plural statuses